CHANGES/8270.bugfix.rst

```diff
@@ -0,0 +1,9 @@
+Fix AsyncResolver to match ThreadedResolver behavior
+-- by :user:`bdraco`.
+
+On system with IPv6 support, the AsyncResolver would not fallback
+to providing A records when AAAA records were not available.
+Additionally, unlike the ThreadedResolver, the AsyncResolver
```

Comment:
Could you double-backtick the identifiers?

Reply:
4f492651b3ae3e693cbbf33916fe05d73e2d046e